Makefile

```diff
@@ -61,27 +65,44 @@ compose_build: build ssl
 compose_stop:
 	$(COMPOSE) kill
 	$(COMPOSE_INTEGRATION) kill
+	$(COMPOSE_PROD) kill
 
 compose_rm:
 	$(COMPOSE) rm -f -v
 	$(COMPOSE_INTEGRATION) rm -f -v
+	$(COMPOSE_PROD) rm -f -v
 
 volumes_rm:
 	docker volume prune -f
 
+static_rm:
+	rm -Rf app/node_modules
+	rm -Rf app/experimenter/legacy-ui/core/node_modules
+	rm -Rf app/experimenter/legacy-ui/rapid/node_modules
+	rm -Rf app/experimenter/legacy-ui/visualization/node_modules
+	rm -Rf app/experimenter/nimbus-ui/node_modules
+	rm -Rf app/experimenter/legacy-ui/assets/
+	rm -Rf app/experimenter/nimbus-ui/build/
```

Comment:
Should this remove the Python cache directories too?

Reply:
Oh we could add one for that, I was just thinking of this as a way to reset all the static paths, I didn't hit a need to remove the python cache dirs since they're ignored by the dockerignore

Reply:
The node_modules are also in the dockerignore, that's why I was wondering what the criteria used here was.

Reply:
No this was for the workflow of resetting the things that run outside the container, a couple times recently people hit a case where those things got out of sync and resetting it all fixed it so I thought it would just be handy to have a reference for it, I've never hit a case where it was necessary to reset py cache dirs